chef/cookbooks/crowbar/templates/default/crowbar-evacuate-host.sh.erb

```diff
@@ -41,17 +41,20 @@ blockmigrate=""
 <% end %>
 if ! nova host-evacuate-live "$host" $blockmigrate; then
     log "Live migration of instances from host $host has failed!"
-    touch $UPGRADEDIR/crowbar-evacuate-host-failed
+    echo 2 > $UPGRADEDIR/crowbar-evacuate-host-failed
     exit 2
 fi
 
-# FIXME: here we need checks for potential errors during the evacuation
-
 while nova list --all-tenants --host "$host" --fields host,status 2>/dev/null | grep -e ACTIVE -e MIGRATING | grep -q "$host"; do
     log "There is still some VM running at $host..."
     sleep 10
 done
 
+if nova list --all-tenants --host "$host" --fields status | grep -q ERROR; then
```

Comment:
What happens if the customer had an instance in error status from some time ago and never cleaned it up? 
Also, having an instance on error does not necessarily mean that the migration failed. I believe that when migrating an instance, if it fails, it should return to `Active`, or at least my tests turned to that.

Maybe we should track the hosts that were `Active` before the migration and confirm that those were migrated or at least they did not fail?